test/seqan3-test.cmake

```diff
@@ -130,6 +130,7 @@ macro (seqan3_require_benchmark)
         gbenchmark_project
         PREFIX gbenchmark_project
         GIT_REPOSITORY "https://github.com/google/benchmark.git"
+        GIT_TAG "e776aa0275e293707b6a0901e0e8d8a8a3679508"
```

Comment:
Can you not give the actual tag instead of the hash? It would make it much easier for readers of the code to know which release we are referring to.